Source/Scene/Cesium3DTile.js

```diff
@@ -61,30 +67,69 @@ define([
         var contentHeader = header.content;
 
 // TODO: For the 3D Tiles spec, we want to change this to basically (x, y, z) to (x, y, z)
-        var b = header.box;
-        var rectangle = new Rectangle(b[0], b[1], b[2], b[3]);
-
-        this._tileBoundingBox = new TileBoundingBox({
-            rectangle : rectangle,
-            minimumHeight : b[4],
-            maximumHeight : b[5]
-        });
-        this._orientedBoundingBox = OrientedBoundingBox.fromRectangle(rectangle, b[4], b[5]);
+        var tileBoundingVolume = header.boundingVolume;
+        if (defined(tileBoundingVolume.box)) {
```

Comment:
Looks good (just that one style comment).  If you want to test a simple case for visualization do something like:
```
box : [7000000, 0, 0,        200000, 0, 0,        0, 200000, 0,        0, 0, 200000]
```
That should make a square box centered at lat/lon (0, 0).